cli/src/funTest/assets/semver4j-ort-result.yml

```diff
@@ -376,7 +376,7 @@ advisor:
       - advisor:
           name: "NexusIQ"
           capabilities:
-            - "VULNERABILITIES"
+          - "VULNERABILITIES"
```

Comment:
BTW, this is interesting in the context of https://github.com/oss-review-toolkit/ort/issues/7219. Actually both indentations are valid YAML (so it's not really a formatting issue that needs fixing). I'm curious to know what change in ORT causes the different serialization... probably a Jackson / SnakeYAML update.

Reply:
> Actually both indentations are valid YAML

I prefer to use what ORT outputs programatically as the single source of truth what is the right intendation. Otherwise, we could not use our own output anymore without reformatting it causing manual effort.

Reply:
> I prefer to use what ORT outputs programatically

My point is: What ORT outputs programmatically seems to have changed (at some point of time). Anyway, I prefer the "new" syntax with less indentation, so I like the change per se.